classes/hab.bbclass

```diff
@@ -116,10 +116,11 @@ do_hab_ivt() {
 
     _image_size_d=$(image_size "${_signing_artifact}")
     _image_size=$(printf "0x%x" ${_image_size_d})
-    # Align to the next multiple of 4096
+    # Align to the next multiple of 12288
+    _align_size=12288
```

Comment:
This is a weird number for the alignment, I am guessing 8K is not enough, so this is 4K more? Is the memory a concern or can we just do 16K to avoid multiples of 3?

Reply:
Yeah, 8K did not cover all existing device trees so I just to the next 4K alignment up. I could do 16K but didn't really find a good reason to increase the memory usage. Why is 16K preferred?

Reply:
No specific reason, but it feels weird to count in multiples of 12K instead of doubling it. But I have no strong arguments, so let's leave it as it is.